src/pages/jaehyeon/Login/Login.js

```diff
@@ -1,5 +1,130 @@
-function LoginJaehyoen() {
-  return <h1>안녕 재현 로그인</h1>;
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import './Login.scss';
+
+function LoginJaehyeon() {
+  const navigate = useNavigate();
+  // let [idValue, setIdValue] = useState('');
+  // let [pwValue, setPwValue] = useState('');
+  const [inputValues, setInputValues] = useState({
+    email: '',
+    password: '',
+  });
+
+  // const handleEmail = event => {
+  //   const { value } = event.target;
+  //   setInputValues({ ...inputValues, email: value });
+  // };
+
+  // const handlePassword = event => {
+  //   const { value } = event.target;
+  //   setInputValues({ ...inputValues, password: value });
+  // }; 위 두 코드 합치면 아래처럼 됨.
+  const handleInput = event => {
+    const { name, value } = event.target;
+    setInputValues({ ...inputValues, [name]: value });
+  }; // 객체의 name이란 키에 value 부여.
+
+  // function saveUserId(e) {
+  //   setIdValue(e.target.value);
+  // }
+  const handleLogin = () => {
+    fetch('http://10.58.0.33:3000/auth/signin', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json;charset=utf-8' },
+      body: JSON.stringify({
+        email: inputValues.email,
+        password: inputValues.password,
+      }),
+    }) //요청
+      .then(response => response.json())
+      .then(data => {
+        // if (data.accessToken) {
+        //   navigate('/Main/jaehyeon');
+        // }
+        data.accessToken
+          ? navigate('/Main/jaehyeon')
+          : alert('ID, PW 확인 요망');
+
+        return window.localStorage.setItem('token', data.accessToken);
+      });
+
+    // navigate('/Main/jaehyeon');
+  };
+  const handleSignUp = () => {
+    fetch('http://10.58.0.33:3000/auth/signup', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json;charset=utf-8' },
+      body: JSON.stringify({
+        email: inputValues.email,
+        password: inputValues.password,
+      }),
+    }) //요청
+      .then(response => response.json())
+      .then(data => console.log(data.message));
+  };
+
+  const isValid =
+    inputValues.email.includes('@') && inputValues.password.length >= 5;
+
+  return (
+    <div className="Login">
```

Comment:
```suggestion
    <div className="login">
```
파스칼 케이스는 컴포넌트 이름에만 젹용되는 컨벤션입니다